packages/lumx-core/src/scss/components/inline-list/_index.scss

```diff
@@ -0,0 +1,15 @@
+@use "sass:list";
+
+/* ==========================================================================
+   Inline list
+   ========================================================================== */
+
+.#{$lumx-base-prefix}-inline-list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+
+    &__item {
+        display: contents;
```

Comment:
Never used if before ^^

Are you aware of this notice from MDN: https://developer.mozilla.org/en-US/docs/Web/CSS/display#display_contents

Reply:
It's the only way I found to not break text overflow